eng/tools/typespec-validation/src/rules/sdk-tspconfig-validation.ts

```diff
@@ -437,6 +437,18 @@ export class TspConfigPythonMgmtPackageDirectorySubRule extends TspconfigEmitter
   }
 }
 
+export class TspConfigPythonMgmtPackageGenerateTestTrueSubRule extends TspconfigEmitterOptionsSubRuleBase {
+  constructor() {
+    super("@azure-tools/typespec-python", "generate-test", true);
+  }
+}
+
+export class TspConfigPythonMgmtPackageGenerateSampleTrueSubRule extends TspconfigEmitterOptionsSubRuleBase {
+  constructor() {
+    super("@azure-tools/typespec-python", "generate-sample", true);
+  }
+}
```

Comment:
Should skip dataplane?

```suggestion
protected skip(_: any, folder: string) {
    return skipForDataPlane(folder);
  }
}
```